editor/js/Menubar.File.js

```diff
@@ -248,6 +248,18 @@ Menubar.File = function ( editor ) {
 
 		} );
 
```

Comment:
Um, and what about the `script` tags in `index.html`? If the two files are not present in the published package, the following lines will cause HTTP 404 errors. 

```javascript
<script src="js/VRControls.js"></script>
<script src="js/VREffect.js"></script>
```